samcli/lib/bootstrap/bootstrap.py

```diff
@@ -29,6 +30,15 @@ def manage_stack(profile, region):
     return bucket_name
 
 
+def get_current_account_id():
+    """Returns account ID based on used AWS credentials."""
+    sts_client = boto3.client("sts")
+    caller_identity = sts_client.get_caller_identity()
+    if "Account" not in caller_identity:
+        raise CredentialsError("Cannot identify account based on configured credentials.")
```

Comment:
Can we make this error message actionable by adding the link: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html ?

Reply:
I am not big fan of adding link in the code TBH. I think `CredentialsError` like explains enough about credentials missing.